inc/switchtec/registers.h

```diff
@@ -406,4 +406,35 @@ struct switchtec_gas {
 	struct pff_csr_regs pff_csr[SWITCHTEC_MAX_PFF_CSR];
 };
 
+struct switchtec_bind_status_in {
```

Comment:
These definitions do not belong in registers.h as they are not GAS registers.

Either put them in switchtec.c along with the functions that use them or if you think there's value in exposing them to the user put them in their own bind.h similar to pmon.h